.gitignore

```diff
@@ -1,2 +1,3 @@
 /**/.DS_Store
```

Comment:
Just a note in general: README file is always useful for other users for your code. I found this blog post quite useful: https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3